compose/ui/ui/src/uikitMain/kotlin/androidx/compose/ui/window/ComposeContainer.uikit.kt

```diff
@@ -96,15 +97,6 @@ internal class ComposeContainer(
     private val windowInfo = WindowInfoImpl().also {
         it.isWindowFocused = true
     }
-    private val mediator: ComposeSceneMediator = ComposeSceneMediator(
```

Comment:
Why did you reverted it?

Reply:
It breaks sample with opening new UIViewController. Because it was no rule to recreate first ComposeSceneMediator after dispose.